src/symbols.ts

```diff
@@ -4,9 +4,21 @@ const hookSymbol = Symbol.for('haunted.hook');
 const updateSymbol = Symbol.for('haunted.update');
 const commitSymbol = Symbol.for('haunted.commit');
 const effectsSymbol = Symbol.for('haunted.effects');
+const layoutEffectsSymbol = Symbol.for('haunted.layoutEffects');
```

Comment:
Were these all `Symbol.for` before? I was thinking they were "private" `Symbol()` but maybe I'm misremembering. 

I don't see a reason to use `Symbol.for` because they're only used internally.

Reply:
These were `Symbol.for` before, but so I let them like that, but it also seemed weird to me, so I'll change that